grafana/dashboards/MonitoringTesting.json

```diff
@@ -0,0 +1,235 @@
+{
```

Comment:
This dashboard will currently get installed in our other grafana instance. I don't think that is what we want. However, we should probably have some sort of deployment for the monitoring stuff right?